app/helpers/application_helper.rb

```diff
@@ -282,16 +282,16 @@ def display_estimated_user_count?
     estimated_user_count > LARGE_USERBASE_THRESHOLD
   end
 
-  # rubocop:disable Rails/OutputSafety
   def admin_config_label(method, content = nil)
-    content ||= raw("<span>#{method.to_s.humanize}</span>")
+    content ||= tag.span method.to_s.humanize
```

Comment:
nitpick non blocking: I noticed there's inconsistency with parenthesis, `tag.span` it's called without it and `tag.a` up in the first file is called with it, and `tag.label` down here is called with them. I vote for parenthesis :D

Reply:
Nice catch :)